charts/apisix/Chart.yaml

```diff
@@ -31,12 +31,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.7.3
+version: 0.7.4
```

Comment:
Since this is a major version update of APISIX, we usually upgrade the version here, and it is recommended to upgrade to 0.8.0

Reply:
Done